Author: Andrey Kutejko <andy128k@gmail.com>
Origin: https://github.com/colin-kiegel/rust-derive-builder/pull/205/
Description: Fix a test faillure following a change on the syn crate.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/block.rs
+++ b/src/block.rs
@@ -67,7 +67,7 @@ mod test {
     use super::*;
 
     #[test]
-    #[should_panic(expected = r#"LexError"#)]
+    #[should_panic(expected = r#"lex error"#)]
     fn block_invalid_token_trees() {
         Block::from_str("let x = 2; { x+1").unwrap();
     }
